bricks/distributed/vector_clock.h

```diff
@@ -34,16 +34,19 @@ class ActsAsVector {
  public:
   explicit ActsAsVector() { storage = std::vector<T>(); };
   explicit ActsAsVector(size_t count) { storage = std::vector<T>(count); };
-  explicit ActsAsVector(ActsAsVector &other) { storage = std::vector<T>(other); };
-  explicit ActsAsVector(const ActsAsVector &other) { storage = std::vector<T>(other.storage); };
+  explicit ActsAsVector(ActsAsVector& other) { storage = std::vector<T>(other); };
```

Comment:
Nit: I believe these can be `= default;`

Reply:
done